packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -1,18 +1,16 @@
 .edit-post-visual-editor {
 	position: relative;
-	padding: 50px 0;
+	padding-top: 50px;
 
 	& .components-button {
 		font-family: $default-font;
 	}
 }
 
 .edit-post-visual-editor .block-editor-writing-flow__click-redirect {
-	// Collapse to minimum height of 50px, to fully occupy editor bottom pad.
-	height: 50px;
+	// Allow the page to be scrolled with the last block in the middle.
+	height: 50vh;
```

Comment:
@jasmussen had a comment on this other simpler PR about using VH https://github.com/WordPress/gutenberg/pull/15165#issuecomment-486573474 as a unit having that varying bottom space is inconsistent as different resolutions. Not sure it applies here though.

Reply:
It partially applies only insofar as we should use the vh unit only when we really want to.

In that other PR, the VH unit added margin to the bottom of the editor, sized in vh. Which meant the margin was different depending on viewport height.

In this PR, and correct me if I'm wrong, the vh unit intends to keep _the line of text you are writing_ at the center of the screen. If that is correct, then it seems to make sense given the typewriter writing experience.